pyanaconda/ui/gui/spokes/storage.py

```diff
@@ -139,7 +139,7 @@ def _get_sw_needs_text(self, required_space, auto_swap):
                          "space, including <b>%(software)s</b> for software and "
                          "<b>%(swap)s</b> for swap space.")
                        % {"product": escape_markup(productName),
-                          "total": escape_markup(str(required_space + auto_swap)),
+                          "total": escape_markup(str(required_space + auto_swap + 0.1 * disk_free)),
```

Comment:
Why 0.1 constant? Could you please make the reason more obvious? Ideally make it `METADATA_SIZE` constant.

Reply:
Added a constant.